hadoop-hdds/docs/content/feature/SCM-HA.zh.md

```diff
@@ -0,0 +1,162 @@
+---
+title: "SCM 高可用"
+weight: 1
+menu:
+   main:
+      parent: 特点
+summary: Storage Container Manager 的 HA 设置可以避免任何单点故障。
+---
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+Ozone包含两个元数据管理节点(用于键管理的 *Ozone Manager* 和用于块空间管理的 *Storage Container management* )和多个存储节点(数据节点)。通过 RAFT 共识算法实现数据在数据节点之间的复制。
+
+<div class="alert alert-warning" role="alert">
+请注意，SCM-HA 尚未准备好在安全环境中进行生产。 安全工作正在进行中，将很快完成。
+</div>
+
+为了避免任何单点故障，元数据管理器节点还应该具有 HA 设置。
+
+Ozone Manager 和 Storage Container Manager 都支持 HA。在这种模式下，内部状态通过 RAFT (使用 Apache Ratis )复制。
+
+本文档解释了 Storage Container Manager (SCM)的 HA 设置，请在[本页]({{< ref "OM-HA" >}})中查看 Ozone Manager (OM)的 HA 设置。虽然它们可以独立地为 HA 进行设置，但可靠的、完全的 HA 设置需要为两个服务启用 HA。
```

Comment:
OM-HA is on a different page， can  check the link？

Reply:
![image](https://user-images.githubusercontent.com/72794035/113093565-c82f0480-9222-11eb-951b-196790cf23e3.png)
Click this button to jump to this page, is it correct?